qp_pacbio/scripts.py

```diff
@@ -111,3 +115,47 @@ def execute(url, job_id, output_dir):
 def finish_qp_pacbio(url, job_id, output_dir):
     """Executes the task given by job_id and puts the output in output_dir"""
     plugin(url, job_id, output_dir)
+
+
+@click.command()
+@click.option("--base", type=click.Path(exists=True), required=True)
+def biom_merge(base):
+    """Merges all PacBio biom tables"""
+    chunk_size = 30
+    for rank in ("none", "per-gene", "ko", "ec", "pathway"):
+        rank = rank + ".biom"
+        tables = glob(f"{base}/bioms/*/{rank}")
+
+        if not tables:
+            continue
+
+        full = None
+        for block in range(0, len(tables), chunk_size):
+            lblock = block + chunk_size
+            chunk = tables[block:lblock]
+            loaded = []
+            for c in chunk:
+                skip = True
+                if _biom.util.is_hdf5_file(c):
+                    skip = False
+                else:
+                    with open(c) as fh:
+                        for i, l in enumerate(fh):
+                            if i >= 1 and l:
+                                skip = False
+                                break
+                if not skip:
+                    temp = _biom.load_table(c)
+                    if temp.shape != (0, 0):
+                        loaded.append(temp)
+
+            if full is None:
+                if len(loaded) == 1:
+                    full = loaded[0]
+                else:
+                    full = loaded[0].concat(loaded[1:])
+            else:
+                full = full.concat(loaded)
+
+            with h5py.File(f"{base}/{rank}", "w") as out:
```

Comment:
Not sure if I'm reading this right, but wouldn't these lines here indicate that you will write `full` to `out` on every iteration of the loop, thus rewriting the same path over and over?